app/assets/stylesheets/boards.scss

```diff
@@ -0,0 +1,34 @@
+.board {
+ 	width: 95%;
+  height: 95%;
+  border: 3px solid #333;
+  margin: 10px auto 10px auto;
+  vertical-align: middle;
+}
+body {
+ 	padding-top: 10%; 
+ 	padding-bottom: 10%;
+}
+td {
+  empty-cells: show;
+}
+table.one, .board {
+  border-collapse: collapse;
+ 	height: 45em;
```

Comment:
Optional: I'm seeing a weird issue where some of the squares are slightly smaller than others. I think this may be due to rounding errors stemming from the fact that you're providing dimensions for the board as a whole and not for each individual cell. Doing the latter and just allowing the table to adjust accordingly seems to take care of it.